lib/entity/author.rb

```diff
@@ -20,6 +20,9 @@ def ==(other)
         name == other.name && biography == other.biography
       end
 
+      def add_nested_in(library)
+      end
```

Comment:
Create class Entity and move this method to it. 
Use this class to inherit in other entity classes. ( expect `Llibrary`)
Also move `include` to this class.